packages/db/src/workspace/pouch/databases.ts

```diff
@@ -209,11 +204,7 @@ export abstract class Databases<C extends Collections> {
         const id = this.generateId(collectionName, resourceInput);
 
         const resource = await this.get(collectionName, id);
-        const {
-          _rev
-        }: {
-          _rev: PouchDB.Core.RevisionId;
-        } = resource ? resource : {};
+        const { _rev = undefined } = resource ? resource : {};
```

Comment:
Nice. I hadn't noticed that {} obviously wouldn't have a `_rev` value. Surprised we haven't seen an error due to this.

Reply:
we didn't need it because the types were looser. now they're tighter, so it's complaining now.